client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfilaStartupConfigurator.kt

```diff
@@ -24,7 +24,7 @@ internal class BackfilaStartupConfigurator @Inject internal constructor(
   private val config: BackfilaClientConfig,
   private val backfilaClient: BackfilaClient,
   @ForBackfila private val moshi: Moshi,
-  @ForBackfila private val backfills: Map<String, KClass<out Backfill<*, *>>>
+  @ForBackfila private val backfills: MutableMap<String, KClass<out Backfill<*, *>>>
```

Comment:
has to be mutable map for mapbinding to work

Reply:
Do we want a point at which it is no longer mutable?

Reply:
the two cases we have are used in very small scopes so I'm not worried